src/Analyzers/Core/Analyzers/AnalyzersResources.resx

```diff
@@ -334,6 +334,18 @@
   <data name="Use_increment_operator" xml:space="preserve">
     <value>Use '++' operator</value>
   </data>
+  <data name="Change_namespace_to_match_folder_structure" xml:space="preserve">
+    <value>Change namespace to match folder structure</value>
+    <comment>{Locked="namespace"} "namespace" is a keyword and should not be localized.</comment>
+  </data>
+  <data name="Namespace_0_does_not_match_folder_structure_expected_1" xml:space="preserve">
+    <value>Namespace "{0}" does not match folder structure, expected "{1}"</value>
+    <comment>{Locked="namespace"} "namespace" is a keyword and should not be localized.</comment>
+  </data>
+  <data name="Namespace_does_not_match_folder_structure" xml:space="preserve">
+    <value>Namespace does not match folder structure</value>
+    <comment>{Locked="namespace"} "namespace" is a keyword and should not be localized.</comment>
+  </data>
```

Comment:
when do you see the last two strings?  why do we need the 3rd string when we have the 2nd?

Reply:
Namespace_does_not_match_folder_structure => title of diagnostic
Namespace_0_does_not_match_folder_structure_expected_1 => description of diagnostic 
Change_namespace_to_match_folder_structure => title of fix to change namespace and fix the diagnostic

Reply:
thanks!